generators/app/templates/src/views/pages/home/home.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { FC } from 'react';
+
+export const Home: FC = () => (
+  <div>
```

Comment:
Is it on purpose that you do not use some of your components etc. inside this page? If so why?

Reply:
I'd like to that in a separate PR, since I reeeeally want to get this shipped 😅